hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/OzoneConfigKeys.java

```diff
@@ -603,6 +603,11 @@ public final class OzoneConfigKeys {
       OZONE_OM_SNAPSHOT_PRUNE_COMPACTION_DAG_DAEMON_RUN_INTERVAL_DEFAULT =
       TimeUnit.HOURS.toMillis(1);
 
+  public static final String OZONE_OM_SNAPSHOT_USE_FULL_DIFF =
+      "ozone.om.snapshot.use.full.diff";
+
+  public static final boolean OZONE_OM_SNAPSHOT_USE_FULL_DIFF_DEFAULT = false;
```

Comment:
Even when this is set to `false`, we can have a client-side option to force a full diff from the CLI. We can do this in another jira.

Reply:
Filed HDDS-8061

Reply:
Yeah its a good idea. Thanks for filing this one.